lib/src/settings.rs

```diff
@@ -267,7 +267,11 @@ impl UserSettings {
     // separate from sign_settings as those two are needed in pretty different
     // places
     pub fn signing_backend(&self) -> Option<String> {
-        self.config.get_string("signing.backend").ok()
+        let backend = self
+            .config
+            .get_string("signing.backend")
```

Comment:
I wonder whether

   .get_string(...)?;

would be clearer than `unwrap_or_default` (assuming it works, of course).

Reply:
I'm not sure it's clearer but it's at least shorter (and not less clear). Good idea.

Reply:
Another option is `.get_string(...).ok().filter(|x| x != "none")`.

I'll unresolve the conversation, but feel free to use it or not.

Reply:
I went with `.ok()?` and a subsequent `.then_some()`